medusa/post_processor.py

```diff
@@ -302,7 +302,7 @@ def _delete(self, files, associated_files=False):
 
         # also delete associated files, works only for 1 file
         if associated_files and len(files) == 1:
-            file_list += self.list_associated_files(files, subfolders=True)
+            file_list += self.list_associated_files(files[0], subfolders=True)
```

Comment:
@medariox nevermind. we pass associated_files=true in `_delete`  to delete existing files in library when we are replacing.